doc/deployment-on-local-cluster.md

```diff
@@ -0,0 +1,59 @@
+# Deployment on Local Cluster
+
+This project allows you to spin up virtualized Kubernetes cluster, using 
+[kubevirtci](https://github.com/kubevirt/kubevirtci) to deploy a local cluster.
+
+## Ephemeral cluster
```

Comment:
This is not consistent with the next heading. "Start an ephemeral cluster"?

Reply:
Done